openapi/index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Symbol Statistic Service API Documentation</title>
+		<!-- needed for adaptive design -->
+		<meta charset="utf-8" />
+		<meta content="width=device-width, initial-scale=1" name="viewport" />
+		<style>
+			body {
+				margin: 0;
+				padding: 0;
+			}
+		</style>
+	</head>
+	<body>
+		<div id="redoc"></div>
+		<script src="https://cdn.jsdelivr.net/npm/redoc@latest/bundles/redoc.standalone.js"></script>
+		<script>
+			Redoc.init(
+				'./openapi.yml',
+				{
+					// suppressWarnings: true,
+					// nativeScrollbars: true,
+					theme: {
+						// colors: {
+						// 	primary: {
+						// 		main: '#5200c6',
+						// 	},
+						// },
+						// typography: {
+						// 	fontFamily: 'Noto Sans, sans-serif',
+						// 	headings: {
+						// 		fontFamily: 'Noto Sans, sans-serif',
+						// 	},
+						// },
+						// breakpoints: {
+						// 	medium: '4500px',
+						// },
+						// spacing: {
+						// 	sectionVertical: 15,
+						// },
+					},
```

Comment:
Should we remove these theme settings if will not be used?

Reply:
the indentation doesn't look right to me